skcosmo/SparseMethods/SparseKPCA.py

```diff
@@ -70,16 +76,15 @@ def fit(self, X, X_sparse=None, y=None):
         :return: Returns the instance itself.
         """
         X = check_array(X, copy=self.copy_X)
-        self._define_kernel_matrix(X, X_sparse)
+        K_sparse_, K_cross_ = self._get_kernel_matrix(X, X_sparse)
```

Comment:
It may read easier to put what you call "K_cross_" first, because it comes first in the Nystroem approximation. 

I still think K_cross_ is not a good name.

Reply:
Ok, I agree